base/server/server.properties

```diff
@@ -34,7 +34,7 @@ white-list=true
 generate-structures=true
 online-mode=true
 max-build-height=256
-level-seed=4492092448794726066
+level-seed=E26
```

Comment:
I'm not sure non-numeric seeds work in server.properties. Do you have any info on that?

Reply:
The [Minecraft wiki](https://minecraft.gamepedia.com/Server.properties#Java_Edition_3) suggests that it does, as it offers several non-numeric strings as examples.

To be safe I tested it just now; using `level-seed=E26` in server.properties, the `seed` command returns `67913`, as expected.